src/directives/class-map.ts

```diff
@@ -15,14 +15,28 @@
 import {AttributePart, directive, Part, PropertyPart} from '../lit-html.js';
 
 // IE11 doesn't support classList on SVG elements, so we emulate it with a Set
+// This shim is also used on first render, to generate a string to commit
+// rather than manipulate classList, to be compatible with SSR.
 class ClassList {
-  element: Element;
+  element: Element|undefined;
+  part: AttributePart|undefined;
```

Comment:
Actually I just realized we don't need to initialize the `ClassList` with the static strings from the committer on first render, because calling `setValue` on the part will result in the classMapped value being concatenated with the static strings automatically, so this was redundant. I believe this redundancy was not caught because semantic-dom-diff de-dupes/orders the classes.